src/components/Button.js

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import { withStyles } from 'material-ui/styles';
+import Button from 'material-ui/Button';
+
+const styles = {
```

Comment:
Please add a `Button.scss` and then `import './Button.css` in your file. You need to move these styles to that file.